### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by goke-epapa**
This PR is to dockerize this base project. The docker container setup uses nginx, fpm and PHP-7. Basic tests have been done to confirm it works fine.

### CODE REVIEW

**Review by forevertga — APPROVED** [review]

**Review by yemexx1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -74,6 +74,31 @@ ant
...
+
+### Installation
+[Mac](https://www.docker.com/docker-mac)
+[Windows](https://www.docker.com/docker-windows)
+[Ubuntu](https://www.docker.com/docker-ubuntu)
+
+### Running Containers
+- Ensure docker is running
+- Change directory to project directory
+- Run the command `docker-compoose up`
```
Comment (tegaphilip) [inline README.md:94]: compose is spelt wrongly

File: README.md [hunk 2]
```diff
@@ -74,6 +74,31 @@ ant
...
+- To rebuild the containers after updates to docker config files, run `docker-composer up --build`
```
Comment (tegaphilip) [inline README.md:95]: compose and not composer

File: README.md (hunk 1 above)
Comment (yemexx1) [reply README.md:94]: Yeah @tegaphilip. I handled it in this [PR](https://github.com/CottaCush/yii2-base-project/pull/9)

Comment (tegaphilip) [reply README.md:94]: Okay @yemexx1 . Thanks.

**Pull request merged**